frontend/views/containers/group-settings/InvitationsTable.vue

```diff
@@ -145,42 +184,47 @@ export default {
     inviteStatusDescription ({
       isAnyoneLink,
       isInviteExpired,
+      isInviteRevoked,
       isAllInviteUsed,
       quantity,
       numberOfResponses
     }) {
       if (isAnyoneLink) return L('{numberOfResponses}/{quantity} used', { numberOfResponses, quantity })
       else if (isAllInviteUsed) return L('Used')
-      else return isInviteExpired ? L('Not used') : L('Not used yet')
+      else if (isInviteRevoked) return L('Not used')
+      else return isInviteExpired || isInviteRevoked ? L('Not used') : L('Not used yet')
     },
     readableExpiryInfo (expiryTime) {
+      const timeLeft = expiryTime - Date.now()
       const MIL = 1000
       const MIL_MIN = 60 * MIL
       const MIL_HR = MIL_MIN * 60
       const MIL_DAY = 24 * MIL_HR
       let remainder
 
-      const days = Math.floor(expiryTime / MIL_DAY)
-      remainder = expiryTime % MIL_DAY
+      const days = Math.floor(timeLeft / MIL_DAY)
+      remainder = timeLeft % MIL_DAY
       const hours = Math.floor(remainder / MIL_HR)
       remainder = remainder % MIL_HR
       const minutes = Math.ceil(remainder / MIL_MIN)
 
       if (days) return L('{days}d {hours}h {minutes}m left', { days, hours, minutes })
       if (hours) return L('{hours}h {minutes}m left', { hours, minutes })
       if (minutes) return L('{minutes}m left', { minutes })
-      return L('Expired')
+      return L('Not used')
```

Comment:
`'Expired'` makes more sense to me and is consistent with the rest of the code.

Reply:
Changed
cc @mmbotelho